docs/store/data-items.md

```diff
@@ -10,20 +10,26 @@ of data and metadata.
 Example function:
 
 ```python
+%%writefile data-prep.py
```

Comment:
This is for jupyter, we don't need it in this doc

Reply:
Just wanted the code to work when pasting it "as is" to the notebook (I guess some users might do it while learning, as I did). I can rewrite it with "code-to-function" instead of the "%%writefile" if it's critical.

Reply:
No need, just tell the user to store the file. This command is Jupyter only and not correct in the doc

Reply:
Fixed